internal/local/environment.go

```diff
@@ -0,0 +1,56 @@
+package local
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/10gen/realm-cli/internal/cli/user"
+)
+
+// Environment contains the name, full filepath, and api keys of a profile
+type Environment struct {
+	Name        string
+	Filepath    string
+	Credentials user.Credentials
+}
+
+// LoadProfileEnvs returns a list of each profile's environment containing name, filepath, and api keys
+func LoadProfileEnvs() ([]Environment, error) {
```

Comment:
[naming/nit] `LoadEnvironments` to match the array of the type this func returns?  I like that [nit] more than suggesting changing the `struct` name to `ProfileEnv`/`ProfileEnvironment`...